packages/elasticsearch-asset-apis/test/api-with-dataFrams-spec.ts

```diff
@@ -0,0 +1,484 @@
+import 'jest-extended';
+import { getESVersion } from 'elasticsearch-store';
+import { LATEST_VERSION, DataTypeConfig } from '@terascope/data-types';
+import { debugLogger, DataEntity, toNumber } from '@terascope/utils';
+import { DataFrame } from '@terascope/data-mate';
+import { EventEmitter } from 'events';
+import {
+    TEST_INDEX_PREFIX,
+    makeClient,
+    cleanupIndex,
+    populateIndex,
+    waitForData,
+    ELASTICSEARCH_VERSION
+} from './helpers';
+import evenSpread from './fixtures/data/even-spread';
+import {
+    createElasticsearchReaderAPI,
+    ESReaderOptions,
+    IDType,
+    InputDateSegments,
+    SlicerDateResults
+} from '../src';
+
+describe('Api', () => {
```

Comment:
This describe block should be:
```suggestion
describe('Reader API with Data Frames', () => {
```